src/EnergyPlus/FanCoilUnits.cc

```diff
@@ -1624,7 +1677,7 @@ namespace FanCoilUnits {
 				// get full load result
 				mdot = FanCoil( FanCoilNum ).MaxColdWaterFlow;
 				SetComponentFlowRate( mdot, FanCoil( FanCoilNum ).ColdControlNode, FanCoil( FanCoilNum ).ColdPlantOutletNode, FanCoil( FanCoilNum ).CWLoopNum, FanCoil( FanCoilNum ).CWLoopSide, FanCoil( FanCoilNum ).CWBranchNum, FanCoil( FanCoilNum ).CWCompNum );
-				Calc4PipeFanCoil( FanCoilNum, ControlledZoneNum, FirstHVACIteration, QUnitOutMaxHC );
+				Calc4PipeFanCoil( FanCoilNum, ControlledZoneNum, FirstHVACIteration, QUnitOutMaxHC, _, ElectricHeaterControl );
```

Comment:
Actually I think I was half way there yesterday when I removed this argument from the Calc4PipeFanCoil call in GeneralRoutines. Since I no longer need it there, I will remove this argument from the FanCoil code as well. I still need some extra logic (to know when the cooling coil is on so the elec heater doesn't also turn on) but I think I can do all that in the Calc routine.
Will make the change and run all my fan coil test files.